crates/miden-testing/tests/auth/rpo_falcon_procedure_acl.rs

```diff
@@ -165,3 +167,202 @@ fn test_rpo_falcon_procedure_acl() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_rpo_falcon_procedure_acl_with_allow_unauthorized_output_notes() -> anyhow::Result<()> {
+    let assembler = TransactionKernel::assembler();
+
+    let component: AccountComponent = AccountMockComponent::new_with_slots(
+        assembler.clone(),
+        AccountStorage::mock_storage_slots(),
+    )
+    .expect("failed to create mock component")
+    .into();
+
+    let mut get_item_proc_root = None;
+    let mut set_item_proc_root = None;
+
+    for module in component.library().module_infos() {
+        for (_, procedure_info) in module.procedures() {
+            match procedure_info.name.as_str() {
+                "get_item" => get_item_proc_root = Some(procedure_info.digest),
+                "set_item" => set_item_proc_root = Some(procedure_info.digest),
+                _ => {},
+            }
+        }
+    }
+
+    let get_item_proc_root = get_item_proc_root.expect("get_item procedure should exist");
+    let set_item_proc_root = set_item_proc_root.expect("set_item procedure should exist");
+    let auth_trigger_procedures = vec![get_item_proc_root, set_item_proc_root];
+
+    let (auth_component, _authenticator) = Auth::ProcedureAcl {
+        auth_trigger_procedures: auth_trigger_procedures.clone(),
+        allow_unauthorized_output_notes: true,
+        allow_unauthorized_input_notes: true,
+    }
+    .build_component();
+
+    let account = AccountBuilder::new([0; 32])
+        .with_auth_component(auth_component)
+        .with_component(component)
+        .account_type(AccountType::RegularAccountUpdatableCode)
+        .storage_mode(AccountStorageMode::Public)
+        .build_existing()?;
```

Comment:
Nit: We could let `MockChainBuilder::add_account_from_builder` handle some of the setup here to make this test a bit shorter, but this also works.